apps/api/src/app/environments-v2/environments.controller.ts

```diff
@@ -35,4 +58,45 @@ export class EnvironmentsController {
       })
     );
   }
+
+  @Post('/publish')
```

Comment:
I don't have any strong opinions, but I wonder if we should make those endpoints more restful so the route will be `/:environmentId/publish` where `environmentId` is the targetEnvironmentId

Reply:
Was also thinking about it, so basically you suggest that source is always Dev by default and could be overriden with another one?